src/applications/mhv/secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -261,7 +267,13 @@ const ComposeForm = props => {
       }
       if (attachments.length) {
         setSaveError(ErrorMessages.ComposeForm.UNABLE_TO_SAVE_DRAFT_ATTACHMENT);
-        setNavigationError(null);
+        setNavigationError({
+          ...ErrorMessages.ComposeForm.UNABLE_TO_SAVE_DRAFT_ATTACHMENT,
+          confirmButtonText:
+            ErrorMessages.ComposeForm.UNABLE_TO_SAVE_DRAFT_ATTACHMENT.editDraft,
+          cancelButtonText:
```

Comment:
For this modal are we using "keep editing" or "continue editing". This looks like it's also throwing off a cypress test.  I would change the constant to either or, or just update the cypress tests.

Reply:
I see that test failing now, UCD-Joel gave me that wording, but it could be an oversight let me check with him first and then circle back to this.